src/plugins/saved_objects/public/_index.scss

```
@@ -0,0 +1 @@
+@import './save_modal';
```

@flash1293 I don't see that the styles from `src/plugins/saved_objects` are imported automatically. Maybe do you know how I can load them?

---

@flash1293 Should be able to help with adding this file to the SASS compiler, but currently this line is wrong. There is no `save_modal.scss` file in this folder. The import needs to point to the index file within that folder instead.
```suggestion
@import './save_modal/index';
```

---

@cchaos Nice catch! thanks.
Since styles can't move to the new platform at the moment, I left its import in [`ui/saved_objects`](https://github.com/elastic/kibana/pull/56383/files#diff-7ee90adb6c62c85f32342044b7d6bce8R1) for now.